addons/Dockerfile

```diff
@@ -28,4 +28,6 @@ RUN \
         cas \
         os-agent
 
+RUN usermod -aG docker vscode
```

Comment:
Let's push that into the command above to reduce layers

Reply:
Done